### PR TITLE
[183] Loops and Numbers

### PR DESCRIPTION
**Pull request by Samju200**
## Task: `looping between two number and print`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/adeyemiJuwon/php/loop.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    if ($x > 100 && $x < 150){
+      echo "$x <br >";
+    }  
+    $x++;
+}
+
+$i = 0;
+while($i < 50){
+    if ($i > 0 && $i % 2 === 0){
+echo "$i <br >";
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/php/loop.php:13]: format this line of code

File: submissions/adeyemiJuwon/php/loop.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?php
+$x = 100;
+while($x <150){
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/php/loop.php:3]: add single space after `<`

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/adeyemiJuwon/php/loop.php (hunk 1 above)
Comment (Samju200) [reply submissions/adeyemiJuwon/php/loop.php]: Okay Done

File: submissions/adeyemiJuwon/php/loop.php (hunk 2 above)
Comment (Samju200) [reply submissions/adeyemiJuwon/php/loop.php]: Okay

File: submissions/adeyemiJuwon/php/loop.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    if ($x > 100 && $x < 150){
+      echo "$x <br>";
+    }  
+    $x++;
+}
+
+$i = 0;
+while($i < 50){
+    if ($i > 0 && $i % 2 === 0){
+echo "$i <br>";
```
Comment (adaugochi) [inline submissions/adeyemiJuwon/php/loop.php:14]: format this line of code

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/adeyemiJuwon/php/loop.php (hunk 3 above)
Comment (Samju200) [reply submissions/adeyemiJuwon/php/loop.php]: @adaugochi Done

**Comment by adaugochi** [comment]
Kindly check line 14 of your code

**Comment by Samju200** [comment]
> Kindly check line 14 of your code

Yes Thank You
Done @adaugochi

**Review by adaugochi — APPROVED** [review]

**Pull request merged**